box_params:
  default_x: 0.0
  default_y: 0.0
  default_size: 50.0
  default_pixel: 128 
  # default_size: 100.0
  # default_pixel: 256
quadnode_params:
  minimum_pixel: 0.390625
  start_Xindex: 64
  start_Yindex: 63
  # start_Xindex: 128
  # start_Yindex: 127
quadtree_params:
  minimum_pixel: 0.390625
  box_height: -1.785
  box_z: 0.00001
  pixel_x: 128
  pixel_y: 128
  # pixel_x: 256
  # pixel_y: 256
  pixel_Xmax: -25
  pixel_Ymax: 25
  # pixel_Xmax: -50
  # pixel_Ymax: 50
  QTsub_topic: "/my_points_raw"
  QTpub_topic: "/detected_boxes"

boxcluster_params:
  correction_eps: 0.07

